lockbox-ios/Action/CopyAction.swift

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+
+struct CopyAction: Action {
+    let text: String
+    let fieldName: String
+}
+
+extension CopyAction: Equatable {
+    static func ==(lhs: CopyAction, rhs: CopyAction) -> Bool {
+        return lhs.text == rhs.text &&
+                lhs.fieldName == rhs.fieldName
+    }
+}
+
+struct CopyDisplayAction: Action {
```

Comment:
It took me a moment to realize this is the action for showing "Copied to clipboard ✓".  It might be worth future us to have a comment about that here.

Reply:
Alternatively, consider a name change to something like CopyConfirmedAction.